### ISSUE TITLE
error after upgrade

### ISSUE THREAD

**Issue by nlomb** [qcubed/framework#951]
just made a fresh install via composer, only kept "configuration.inc.php" and demo database,
codegen, open form drafts (person) and got this:
```
 Error in PHP Script /resources/project/forms/person_list.php
syntax error, unexpected 'foreach' (T_FOREACH)

Error Type: E_PARSE

Source File: /vendor/qcubed/framework/includes/base_controls/QControlProxy.class.php Line: 178

Line 173:    		}
Line 174:    		
Line 175:    		public function RenderAsScript($strEventType='QClickEvent') {
Line 176:     +			$objActions 	= $this->GetAllActions($strEventType);
Line 177:     +			$strToReturn 	= '';
Line 178:     +			foreach ($objActions as $objAction) {
Line 179:     +				$strToReturn .= $objAction->RenderScript($this);
Line 180:     +			}
Line 181:     +			return $strToReturn;
Line 182:     +		}
Line 183:    
```

remove the "+" signs in lines 176 -182 in file "/vendor/qcubed/framework/includes/base_controls/QControlProxy.class.php"

**Comment by spekary** [comment]
Fixed in #952

**Comment by nlomb** [comment]
thanks
